packages/utilities/psammead-storybook-helpers/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 7.0.1 | [PR#XXX](https://github.com/bbc/psammead/pull/XXX) Update chinese locales |
+| 7.0.0 | [PR#2404](https://github.com/bbc/psammead/pull/2404) replace inputProvider and dirDecorator with withServicesInput |
```

Comment:
There's a duplication of ` | 7.0.0 | [PR#2404] ` and ` | 7.0.1 | [PR#XXX] ` doesn't have a PR number